helm/prometheus-rules/templates/alerting-rules/aws.workload-cluster.rules.yml

```diff
@@ -24,6 +24,7 @@ spec:
         cancel_if_cluster_status_deleting: "true"
         cancel_if_cluster_status_updating: "true"
         cancel_if_outside_working_hours: "true"
+        cancel_if_has_no_workers: "true"
```

Comment:
I'm being pedantic here but can we call this cancel_if_cluster_has_no_workers ? Because this could apply to node pools also

Reply:
ok then please approve the renaming here https://github.com/giantswarm/prometheus-meta-operator/pull/779

Reply:
It's done :)

Reply:
Now I need to release :) https://github.com/giantswarm/prometheus-meta-operator/pull/780

Reply:
thanks Quentin